.cirrus.yml

```diff
@@ -553,7 +564,7 @@ windows_smoke_test_task:
 local_integration_test_task: &local_integration_test_task
     # Integration-test task name convention:
     # <int.|sys.> <podman|remote> <Distro NV> <root|rootless>
-    name: &std_name_fmt "$TEST_FLAVOR $PODBIN_NAME $DISTRO_NV $PRIV_NAME $TEST_ENVIRON"
+    name: &std_name_fmt "$TEST_FLAVOR $PODBIN_NAME $DISTRO_NV $PRIV_NAME $TEST_ENVIRON $CI_DESIRED_DATABASE"
```

Comment:
UGH. Cirrus variable expansion is weird. You may need to curlybrace it: `${CI_DESIRED_DATABASE}`